en/docs/get-started/open-packages.md

```diff
@@ -1,16 +1,16 @@
 # Open Ballerina packages
 
-You can work on a single [Ballerina package](https://ballerina.io/learn/package-references/) at a time or use [Multi-Root workspaces](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) of VS Code to work on serverl related Ballerina Packages at the same time. 
+You can work on a single [Ballerina package](https://ballerina.io/learn/package-references/) at a time or use [multi-Root workspaces](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) of the VS Code editor to work on serveral related Ballerina packages at the same time. 
 
 ## Open a Ballerina package
 
-Use one of the following options to open a Ballerina package:
+Use one of the following options to open a Ballerina package on the VS Code editor:
 
 - From the command line, execute `code <path-to-package-dir>`.
-- From VS Code, click **File > Open Folder ...**.
+- From the VS Code editor, click **File > Open Folder ...**.
```

Comment:
Do we need to say `editor` always? Even vs code docs doesn't followed it with the word `editor`

https://code.visualstudio.com/docs/editor/editingevolved